<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="history_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;

</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>190</number>
   </attribute>
   <column>
    <property name="text">
     <string>Кол-во попыток</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сложность</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="clear_history">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Очистить историю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_farm">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>100</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Обратно на ферму</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_history">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>1171</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background_menu">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>-30</y>
     <width>1261</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="difficulty_select">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>Вся история</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Легкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Средний</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сложный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень сложный</string>
    </property>
   </item>
  </widget>
  <zorder>background_menu</zorder>
  <zorder>background_history</zorder>
  <zorder>history_tableWidget</zorder>
  <zorder>clear_history</zorder>
  <zorder>back_to_farm</zorder>
  <zorder>difficulty_select</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
